website/docs/guides/serverless/about.md

```diff
@@ -7,8 +7,16 @@ Currently the only supported serverless deployment of Benthos is for
 [AWS Lambda][lambda], if you are interested in other platforms please
 [raise an issue](https://github.com/benthosdev/benthos/issues).
 
+:::info Want to keep things boring?
+Get deployed quickly with this production-ready project template: [makenew/serverless-benthos].
+
+Or, if you need a deployable build with custom plugins: [makenew/benthos-plugin].
```

Comment:
And maybe something like:

> Rather than bundle the distribution and configs yourself check out [makenew/serverless-benthos], which makes quick work of deploying a Benthos serverless project on AWS Lambda.